tests/lib/Uniswap/data.ts

```diff
@@ -0,0 +1,40 @@
+import { ACTION_ENUM, CHAIN_ID } from '../../../src/enums';
+import { IProtocolTestingData } from '../../../src/types/chunks/ITestingData';
+
+export enum UNISWAP_VERSIONS {
+  V2 = 'v2',
+  V3 = 'v3'
+}
+
+export const uniswapData: IProtocolTestingData = {
```

Comment:
Lets add some more test cases please
Ideally atleast 10 transactions per protocol!

Try to add transactions where the path length > 2